packages/donate-button-beta/src/public/index.html

```diff
@@ -254,10 +254,12 @@ <h1 style="color: white">COVID-19 Pandemic Response: India</h1>
           selector: "#incomemovement-donate-button",
           options: {
             nonprofitSlug: "incomemovement",
-            primaryColor: "#1AAFA7",
-            showInitialMessage: false,
+            primaryColor: "#00A37F",
+            showInitialMessage: true,
+            showInputButtons: true,
```

Comment:
```suggestion
            showInitialMessage: false,
            showInputButtons: false,
```
Before land, can you change both of these to false? Then I can send https://embeds.every.org/0.3 link to Income Movement :D